content/OTP/Specifications/OTP_validation_protocol.adoc

```diff
@@ -6,9 +6,13 @@ All requests are HTTP GET requests. As such, all parameters must be
 properly URL encoded.  In particular, some base64 characters (such as
 "+") in the value fields needs to be escaped.
 
+Each request sent to the server can be signed. To ensure that the
+server rejects any requests that have been tampered with, clients
+should sign each request they make to the server.
```

Comment:
There's a mix throughout the document of informal-style "must" and "should", and RFC2119-style "MUST" and "SHOULD". For consistency I think we should pick just one.

Reply:
I will change this to match the rest of the document.